nsv13/code/modules/overmap/weapons/projectiles_fx.dm

```diff
@@ -110,13 +109,36 @@ Misc projectile types, effects, think of this as the special FX file.
 	name = "plasma torpedo"
 	speed = 2.75
 	valid_angle = 150
-	homing_turn_speed = 30
-	damage = 200
+	homing_turn_speed = 35
+	damage = 240
 	range = 250
-	armor = list("overmap_light" = 60, "overmap_heavy" = 10)
+	armor = list("overmap_light" = 20, "overmap_heavy" = 10)
 	flag = "overmap_heavy"
 	impact_effect_type = /obj/effect/temp_visual/impact_effect/torpedo
 	spread = 5 //Helps them not get insta-bonked when launching
+/obj/item/projectile/guided_munition/torpedo/shredder
+	icon_state = "torpedo_shredder"
+	name = "plasma charge"
+	damage = 240
+	armour_penetration = 20
+
+/obj/item/projectile/guided_munition/torpedo/decoy
+	icon_state = "torpedo"
+	damage = 0
+	obj_integrity = 200
+	max_integrity = 200
+
+/obj/item/projectile/guided_munition/torpedo/nuclear
+	icon_state = "torpedo_nuke"
+	name = "thermonuclear cruise missile"
```

Comment:
if it's of any importance, it's only a cruise missile if it's for a terrestrial target 😎